### PR TITLE
Extending CHANGES file

### PR DESCRIPTION
**Pull request by dune73**

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -2,14 +2,22 @@
...
- * Panic with error 500 if the crs-setup.conf file is not loaded.
+ * Generic mechanism to support application specific rule exclusions
+   (Chaim Sanders)
+ * Initial Wordpress rule exclusions (Walter Hop)
+ * Initial Drupal rule exclusions (Christian Folini, @emphazer)
+ * Cleanup of reputation checks / persistent blocking 
+   (Christina Folini / Walter Hop)
+ * Shortened overly long RegExes to work on Apache 2.2
+ * Support for HTTP/2 protocol in request line
+ * Updated list of webscanners
```
Comment (csanders-git) [inline CHANGES]: I'm being picky but perhaps just add malicious webscanners

**Comment by csanders-git** [comment]
Other than my comment it's good except we didn't talk about the fix for Apache 2.2

File: CHANGES (hunk 1 above)
Comment (dune73) [inline CHANGES]: Yes, that's more precise. Thanks.

**Comment by lifeforms** [comment]
@dune73 Yes, that's fixed. Just not the Apache < 2.4.11 issue. But as @emphazer said, my `cont.py` script from #610 can be used to join multiline rules into one rules, so we could put that in `util` and mention it in the known bugs.

**Comment by dune73** [comment]
Good plan @lifeforms. Would you mind describing it for KNOWN_BUGS? Either here or in a separate PR.

**Comment by lifeforms** [comment]
@dune73 I'm tied up at the moment, but I'll add it to your PR #612 tonight.

**Comment by emphazer** [comment]
@lifeforms i said: _theoretical_ ;-)

**Comment by lifeforms** [comment]
@emphazer Ow :( Did you actually try it though? I have no old Apaches here so it would take me a lot of time to set it up, but it would be kind of neat if we could officially publish a workaround.

**Comment by dune73** [comment]
I've got a 2.4.7 laying around in my lab. I could fire it up tonight.

**Comment by emphazer** [comment]
@lifeforms no, i didnt test it. it was more like a joke because i know that you guys dont like it if someone modifies the ruleset. 

the call is still open https://bugzilla.redhat.com/show_bug.cgi?id=1378946

**Comment by dune73** [comment]
Yeah. The problem with this bug is you have to modify it - or upgrade apache and if that was an option, they would certainly do so.

**Comment by emphazer** [comment]
well, found the following working solution

`sed -i '{s@^[ \t]\+@@g;s/[\t ]*$//;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;N;s/,\\\n/,/g;}'  rules/*.conf`

it make this rule for example:
```
SecRule ARGS_NAMES "." \
"phase:request,\
id:921170,\
rev:'2',\
ver:'OWASP_CRS/3.0.0',\
pass,\
nolog,\
tag:'application-multi',\
tag:'platform-multi',\
tag:'attack-protocol',\
tag:'paranoia-level/3',\
tag:'CAPEC-460',\
setvar:'TX.paramcounter_%{MATCHED_VAR_NAME}=+1'"
```


looks like that 

```
SecRule ARGS_NAMES "." \
"phase:request,id:921170,rev:'2',ver:'OWASP_CRS/3.0.0',pass,nolog,tag:'application-multi',tag:'platform-multi',tag:'attack-protocol',tag:'paranoia-level/3',tag:'CAPEC-460',setvar:'TX.paramcounter_%{MATCHED_VAR_NAME}=+1'"
```

**Comment by dune73** [comment]
I see this working, but adding it to the KNOWN_BUGS makes the workaround look quirky. I'd say we leave it as is.

**Comment by lifeforms** [comment]
Added some more stuffs.

**Comment by dune73** [comment]
So we're done here as well. I'm merging.

**Pull request merged**